lib/msf/core/exploit/file_dropper.rb

```diff
@@ -126,5 +96,105 @@ def cleanup
     end
 
   end
+
+  private
+
+  # See if +path+ exists on the remote system and is a regular file
+  #
+  # @param path [String] Remote filename to check
+  # @return [TrueClass] If the file exists
+  # @return [FalseClass] If the file doesn't exist
+  def file_dropper_file_exist?(session, path)
+    if session.platform =~ /win/
+      normalized = file_dropper_win_file(path)
+    else
+      normalized = path
+    end
+
+    if session.type == 'meterpreter'
+      stat = session.fs.file.stat(normalized) rescue nil
+      return false unless stat
+      stat.file?
+    else
+      if session.platform =~ /win/
+        f = shell_command_token("cmd.exe /C IF exist \"#{normalized}\" ( echo true )")
+        if f =~ /true/
+          f = shell_command_token("cmd.exe /C IF exist \"#{normalized}\\\\\" ( echo false ) ELSE ( echo true )")
+        end
+      else
+        f = session.shell_command_token("test -f \"#{normalized}\" && echo true")
+      end
+
+      return false if f.nil? || f.empty?
+      return false unless f =~ /true/
+      true
+    end
+  end
+
+  # Sends a file deletion command to the remote +session+
+  #
+  # @param [String] file The file to delete
+  # @return [TrueClass] the delete command has been executed in the remote machine
```

Comment:
@jvazquez-r7 If you don't mind, I will go ahead and modify TrueClass/FalseClass to just Boolean. As @jlee-r7 suggested.